dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/HostsUpdater.java

```diff
@@ -107,28 +114,27 @@ public HostStatusTracker refreshHosts() {
             if (hostFromHostSupplier.isUp()) {
                 Host hostFromTokenMapSupplier = allHostSetFromTokenMapSupplier.get(hostFromHostSupplier);
 
-                hostsUpFromHostSupplier.add(new Host(hostFromHostSupplier.getHostName(), hostFromHostSupplier.getIpAddress(),
-                        hostFromTokenMapSupplier.getPort(), hostFromTokenMapSupplier.getSecurePort(), hostFromTokenMapSupplier.getRack(),
-                        hostFromTokenMapSupplier.getDatacenter(), Host.Status.Up, hostFromTokenMapSupplier.getHashtag(),
-                        hostFromTokenMapSupplier.getPassword()));
+                hostsUpFromHostSupplier.add(new HostBuilder().setHostname(hostFromHostSupplier.getHostName())
```

Comment:
nit - add a convenience method that creates a Host from a given Host to avoid duplicating the code to copy it over here and both places below

Reply:
good point. updated!